go/ct/gen/state.go

```diff
@@ -227,6 +229,16 @@ func (g *StateGenerator) BindIsStorageCold(key Variable) {
 	g.storageGen.BindCold(key)
 }
 
+// BindIsTransientSet wraps TransientGenerator.BindSet.
```

Comment:
nit: this document string is useless since `TransientGenerator.BindSet` is not documented either;